src/DotNetWorker.Core/Definition/FunctionDefinition.cs

```diff
@@ -44,5 +45,10 @@ public abstract class FunctionDefinition
         /// Gets the output binding metadata.
         /// </summary>
         public abstract IImmutableDictionary<string, BindingMetadata> OutputBindings { get; }
+
+        /// <summary>
+        /// Gets if a the function is binding to a cancellation token.
+        /// </summary>
+        public abstract bool BindsToCancellationToken { get; }
```

Comment:
Would this be a breaking change as this type is public ?

Reply:
Good question, I'm not sure! What's the best way to double check this?

Reply:
1 1  A M P S T 🏰

Reply:
Removed this prop so no longer relevant